src/Symfony/Component/Cache/Tests/Adapter/PredisTagAwareRedisClusterAdapterTest.php

```diff
@@ -1,35 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\Cache\Tests\Adapter;
-
-use Psr\Cache\CacheItemPoolInterface;
-use Symfony\Component\Cache\Adapter\RedisTagAwareAdapter;
-use Symfony\Component\Cache\Tests\Traits\TagAwareTestTrait;
-
-class PredisTagAwareRedisClusterAdapterTest extends PredisRedisClusterAdapterTest
```

Comment:
should we remove that test entirely ? Or only change the Predis cluster connection being used ?

Reply:
There's another test case using PredisCluster - this one is for RedisCluster, which is not supported.